style.css

```diff
@@ -0,0 +1,775 @@
+}
+
+.stories__foto {
+  width: 100%;
+}
+
+.stories__data {
+  text-align: right;
+  font-weight: bold;
+  opacity: 0.3;
+}
+
+.hashtag__foto {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.hashtag__img {
+  width: 14.3%;
+}
+
+.hashtag__img:nth-of-type(1n+2) {
+  margin-left: 0;
+  padding: 0;
+}
+
+.info {
+  display: flex;
+  flex-flow: row wrap;
+  margin: 0;
+  padding: 0;
+  background: #000;
+  color: #fff;
+}
+
+@media (max-width:1070px) {
+  body {
+    padding: 0 4.6%;
+  }
+
+  .navigation {
+    margin-bottom: 34px;
+  }
+
+  .navigation__img {
+    width: 61px;
+    height: 64px;
+    margin-right: 3%;
+    padding-top: 3px;
+  }
+
+  .navigation__banner {
+    padding-top: 26px;
+  }
+
+  .navigation__submenu {
+    margin-top: 3px;
+    margin-right: 0.5%;
+  }
+
+  .navigation__link {
+    margin-right: 7.5%;
+  }
+
+  .navigation__form {
+    padding-top: 8px;
+    margin-right: 2.4%;
+  }
+
+  .navigation__search {
+    width: 8.6%;
+  }
+
+  .catalog__wrapper {
+    height: 253px;
+    padding: 75px 0 0 53px;
+    margin-bottom: 60px;
+  }
+
+  .catalog__title {
+    padding-bottom: 5px;
+  }
+
+  .catalog__arrow {
+    width: 6.3%;
+    padding-left: 10px;
+  }
+
+  .catalog__subtitle {
+    margin-bottom: 27px;
+  }
+
+  .catalog__menu {
+    margin-bottom: 44px;
+  }
+
+  .catalog__item {
+    min-width: 0;
+  }
+
+  .catalog__round_grey,
+  .catalog__round_red,
+  .catalog__round_yellow,
+  .catalog__round_blue {
+    width: 6.33px;
+  }
+
+  .catalog__round_black,
+  .catalog__round_peach,
+  .catalog__round_wine,
+  .catalog__round_azure {
+    width: 8px;
+  }
+
+  .catalog__img {
+    width: 170px;
+    margin-bottom: 32px;
+  }
+
+  .catalog__scale {
+    width: 26%;
+  }
+}
+
+@media (max-width: 991px) {
+  body {
+    padding: 0;
+  }
+
+  .navigation {
+    margin-bottom: 17px;
+  }
+
+  .navigation__logo {
+    display: block;
+    margin: 0;
+    position: absolute;
+    top: 41.5px;
+    left: 5.6%;
+    width: 20.5%;
+}
+
+  .navigation__banner {
+    display: none;
+  }
+
+  .navigation__burger {
+    display: block;
+    margin: 0;
+    position: absolute;
+    top: 46.5px;
+    right: 5.7%;
+  }
+
+  .navigation__burger div {
+    background: #000;
+    width: 61px;
+    height: 9px;
+    margin-bottom: 7.6px;
+  }
+
+  .catalog__wrapper {
+    height: 415px;
+    padding: 15.6% 0 0 5.9%;
+    background-position-x: -129px;
+    background-size: 200%;
+    margin-bottom: 62px;
+  }
+
+  .catalog__title {
+    font-size: 50px;
+    line-height: 48px;
+    width: 45%;
+    padding-bottom: 49px;
+  }
+
+  .catalog__shop a {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .catalog__subtitle {
+    font-size: 30px;
+    margin-right: 2%;
+  }
+
+  .catalog__menu {
+    justify-content: normal;
+  }
+
+  .catalog__item {
+    width: 34%;
+    height: 352px;
+  }
+
+  .catalog__item_1 {
+    margin-right: 0;
+  }
+
+  .catalog__item:nth-child(n+3) { display: none; }
+  .catalog__item:first-child { 
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Project.Axion/pullRequest?prid=5456827)